src/common.js

```diff
@@ -17,7 +29,8 @@ var config = {
   verbose: false,
   noglob: false,
   globOptions: {},
-  maxdepth: 255
+  maxdepth: 255,
+  nodepath: false
```

Comment:
Could you add a trailing comma? We don't enforce it via linting, but it's something we're trying to move toward to keep diffs simple.